Funcky.DiscriminatedUnion.SourceGeneration/Funcky.DiscriminatedUnion.SourceGeneration.csproj

```diff
@@ -40,6 +41,6 @@
     <ItemGroup>
         <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.3" PrivateAssets="all" />
         <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.0.1" />
-        <PackageReference Include="IsExternalInit" Version="1.0.1" PrivateAssets="all" />
+        <PackageReference Include="Meziantou.Polyfill" Version="1.0.39" PrivateAssets="all" />
```

Comment:
Careful: Removing IsExternalInit is usually a binary breaking change (not here, because this is not intended to be used as a library)

In Funcky we use [Polysharp](https://github.com/Sergio0694/PolySharp), so it might make sense to also use that here. Are there any advantages to using Meziantou.Polyfill?

Reply:
It's the same thing as far as I understand. So I swapped to PolySharp, and everything works the same.